types/client/adCustomConfig.ts

```diff
@@ -0,0 +1,6 @@
+export type AdCustomConfig = {
```

Comment:
Well, now I think it is better to move all ad types in one file - `types/client/ad`. I will appreciate if you do that.

Reply:
Ok, I'll do that.